app/controllers/index.js

```diff
@@ -13,19 +13,25 @@ export default class extends Controller {
 
   @alias('model') days;
 
+  get isDebug() {
+    return !this.fastboot.isFastBoot && window.location.search.includes('debug');
+  }
+
   _setNow() {
     if (ENV.APP.shouldForceDayOne) {
       // Use conf Day 1 for development
       let time = moment().utcOffset(ENV.APP.UTC_OFFSET).format('HH:mm:ss');
-      this.now = moment(`2019-03-19T${time}${ENV.APP.UTC_OFFSET}`).format();
+      this.now = `2019-03-19T${time}${ENV.APP.UTC_OFFSET}`;
     } else {
       // Use real date and time for non-dev environments
       this.now = moment().utcOffset(ENV.APP.UTC_OFFSET).format();
     }
 
-    if (!ENV.APP.shouldUpdateTime || this.fastboot.isFastBoot) { return; }
+    if (!ENV.APP.shouldUpdateTime || this.fastboot.isFastBoot || this.isDebug) {
+      return;
+    }
 
-    later(this, function() {
+    this._timeout = window.setTimeout(() => {
```

Comment:
I don't think we're using `this._timeout` any longer.

We should likely either a) check for `this.isDestroying` in the callback here, or b) cancel the scheduled timeout.